### ISSUE TITLE
BUG - [LOW] - [All] - Accessibility 5103 - Buttons on the "How to submit evidence" page don't go to the top of the following page with screen readers on

### ISSUE THREAD

**Issue by digitalken** [department-of-veterans-affairs/va-mobile-app#12227]
## What happened?

From the "How to submit evidence page", tapping "Select a file" and "Take or select photos" buttons with the screen reader on, does not focus the user to the top of the next page.  Focus will land on the submit buttons or elsewhere on the page (as seen in the screen shots below.)
## Specs:

- Device: iPhone, Android
- OS: iOS, Android
- User Account: +228. (Cory Sohrakoff can set claims to need 5103 flow, if needed.)
- Accessibility State: Screen reader ON

Slack thread for adding user data:  https://dsva.slack.com/archives/C09B6NPTBC4

## Steps to Reproduce

1. With the screen reader on, navigate to the "How to submit evidence" page in the 5103 flow.
2. Tap on the "Select a file" button and notice that user lands on either text at the bottom of the following page or the submit button, depending on OS.
3. Tap on the "Take or select photos" button and notice that user lands take or select photos button on the following page and not the top of the page.
## Desired behavior

When transitioning to the next page, user's focus should land at the top of the page with the screen reader on.
## Acceptance Criteria

Make the focus of the screen reader land at the top of the following pages.

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact:  Low 
- Frequency: Low 

- 3 - Low

## Linked to Story

PR:  https://github.com/department-of-veterans-affairs/va-mobile-app/pull/12205
## Screen shot(s) and additional information

<img width="1010" height="833" alt="Image" src="https://github.com/user-attachments/assets/86a91b42-723b-4272-a2f5-a21ed92affe3" />

<img width="1038" height="830" alt="Image" src="https://github.com/user-attachments/assets/448186be-6064-4803-b751-9c8025bc4de4" />

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  

</details>

## Ticket Checklist
- [ ] Steps to reproduce are defined
- [ ] Desired behavior is added
- [ ] Labels added (front-end, back-end, global, Health, relevant feature, accessibility, etc)
- [ ] Estimate of 1 added (for front-end tickets)
- [ ] Added either to the relevant feature epic (if found during new feature implementation), or the relevant team's bug epic ([Global](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8302), [Health & Benefits](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8303), [API](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8304), [QART](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8305))
